features/admin.actions.v1/api/change-action-status.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AsgardeoSPAClient, HttpClientInstance } from "@asgardeo/auth-react";
+import { store } from "@wso2is/admin.core.v1/store";
+import { IdentityAppsApiException } from "@wso2is/core/exceptions";
+import { HttpMethods } from "@wso2is/core/models";
+import { AxiosError, AxiosRequestConfig, AxiosResponse } from "axios";
+import { ActionsConstants } from "../constants/actions-constants";
+import { ActionBasicResponseInterface } from "../models/actions";
+
+/**
+ * Get an axios instance.
+ */
+const httpClient: HttpClientInstance = AsgardeoSPAClient.getInstance()
+    .httpRequest.bind(AsgardeoSPAClient.getInstance());
+
+/**
+ * Activate/Deactivate an Action under a given action type and id.
+ *
+ * @param actionType - Action Type.
+ * @param actionId - Action Id.
+ * @param status - activate/deactivate option.
+ * @returns Promise containing the response.
+ * @throws Throws an IdentityAppsApiException if the request fails.
+ */
+const changeActionStatus = (actionType: string, actionId: string, status: string):
+Promise<ActionBasicResponseInterface> => {
```

Comment:
Let's improve the formatting here.
```suggestion
const changeActionStatus = (
    actionType: string,
    actionId: string,
    status: string
): Promise<ActionBasicResponseInterface> => {
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/5e3564e0797c09f4f20b71837b928bc9fa57c847